### ISSUE TITLE
Improve networks.wamr.normalize

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/crmprtd#147]
These things noted in passing while solving another problem:

- Function `get_one_of` has probable problems as noted below.
- In body of `normalize`, inside loop over input rows, the file `networks/wamr/station_substitutions.yaml` is opened on each loop iteration. It should be opened once, outside the loop.

```python
def get_one_of(elements):
    for e in elements:
        # FIXME: Condition should probably be `e is not None`
        if e:
            return e
    # FIXME: {e} should probably be {elements}
    raise ValueError(f"No elements of {e} have a truthy value")
```

**Comment by jameshiebert** [comment]
You're right on the 2nd FIXME. I think that the 1st FIXME is mistaken. Based on the context, this function is looking for "truthy" values, so it wants to eliminate `None`s, but also empty strings.
